cmd/bbolt/command_buckets.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	bolt "go.etcd.io/bbolt"
+)
+
+func newBucketsCommand() *cobra.Command {
+	var bucketsCmd = &cobra.Command{
+		Use:   "buckets <path>",
+		Short: "print a list of buckets",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return bucketsFunc(cmd, args[0])
+		},
+	}
+	return bucketsCmd
+}
+
+func bucketsFunc(cmd *cobra.Command, srcDBPath string) error {
+	// Required database path.
+	if srcDBPath == "" {
+		return ErrPathRequired
+		// Verify if the specified database file exists.
+	} else if _, err := checkSourceDBPath(srcDBPath); err != nil {
+		return err
+	}
+
+	// Open database.
+	db, err := bolt.Open(srcDBPath, 0600, &bolt.Options{ReadOnly: true})
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	// Print the list of buckets in the database.
+	return db.View(func(tx *bolt.Tx) error {
+		return tx.ForEach(func(name []byte, _ *bolt.Bucket) error {
+			fmt.Fprintln(cmd.OutOrStdout(), string(name))
```

Comment:
I agree that `cmd.OutOrStdout()` is a little better than `os.Stdout`, but please keep it consistent with other commands. We can replace them with `cmd.OutOrStdout()` in separate PR.

Reply:
cmd.OutOrStdout() is all ready used in the cobra style converted files already
command_check.go line 44, 65, 70
command_get.go line 46, 92
command_info.go line 47
command_version.go line 18, 19, 20

Reply:
I see that it's only used in `command_check.go`. I should have raised this comment in [PR 723](https://github.com/etcd-io/bbolt/pull/723).

Overall, not a big deal. Feel free to keep it as it's

Reply:
I must have changed the over files in my copy above when I was testing the outputs.
As I had trouble capturing the output when I had converted to the cobra style without the cmd.OutOrStdout() function